package.json

```diff
@@ -10,8 +10,9 @@
     "scripts": {
         "compile": "hardhat compile",
         "test": "hardhat test",
+        "test:ci": "cross-env COVERAGE=true hardhat test",
         "deploy": "hardhat ignition deploy ignition/modules/$npm_config_module.ts --network $npm_config_network",
-        "coverage": "hardhat coverage",
+        "coverage": "cross-env COVERAGE=true hardhat coverage",
```

Comment:
we need to minimize what tests coverage needs to have access to. all the gas optimization stuff is too heavy to run within CI, so setting some conditionals for it to run smartly will ensure this doesn't weigh down future work and spend unneeded time waiting for coverage to run